src/locale/locales/tr/messages.po

```diff
@@ -9812,11 +9812,11 @@ msgstr ""
 
 #: src/components/dms/MessagesNUX.tsx:119
 #~ msgid "You can change this at any time."
-#~ msgstr ""
+#~ msgstr "Bunu istediğiniz zaman değiştirebilirsiniz."
 
 #: src/screens/Messages/Settings.tsx:105
 msgid "You can continue ongoing conversations regardless of which setting you choose."
-msgstr ""
+msgstr "Hangi ayarı seçerseniz seçin devam eden görüşmelere devam edebilirsiniz."
```

Comment:
Looks better with less repetition, please feel free to ignore if you don't think the same.
```suggestion
msgstr "Hangi ayarı seçerseniz seçin devam eden görüşmeleri sürdürebilirsiniz."
```